test/kubemark/common.sh

```diff
@@ -49,8 +49,13 @@ export RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 # arguments:
 # $@: all stuff that goes after 'gcloud compute '
 function run-gcloud-compute-with-retries {
+  echo "" > /tmp/gcloud_retries
   for attempt in $(seq 1 ${RETRIES}); do
-    if ! gcloud compute $@; then
```

Comment:
I'm not sure about this change. This will make 'start-kubemark' executions work, even if previous run was not cleaned up correctly. I think it's worse than failing in case of some stalled gcloud call. This is not very strong opinion, so feel free to disagree and keep it. If you do append some hash of command being run after `gcloud_retries` to make it thread safe.